pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -537,16 +544,17 @@ private static double SumIntensities(MsDataSpectrum[] spectra, double mz, int[]
 
         private void GetMaxMzIntensity(out double maxMz, out double maxIntensity)
         {
-            var fullScans = _msDataFileScanHelper.GetFilteredScans();
+            var fullScans = _msDataFileScanHelper.GetFilteredScans(out _, out _);

```

Comment:
Does this code want the scans filtered by ion mobility, or not?
If it wants the unfiltered scans, then shouldn't it be using "MsDataSpectra" directly, instead of calling "GetFilteredScans"?
If it wants the filtered scans, then doesn't it need to do something with the two out parameters that it's ignoring here?

The way the code is now, I think the behavior is going to end up being different depending on whether the data is stored as 3 arrays or not.

Reply:
At this point it just wants to know the mz and intensity range of scans have data that passes within the IM filter, but doesn't need to know the IM range.